.github/stale.yml

```
@@ -0,0 +1,41 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 0.25
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 0.25
```

Change to half day to conform to [zer0db/staletest-01](https://github.com/zer0db/staletest-01)
```suggestion
daysUntilClose: 0.5
```